content/execution_model.tex

```diff
@@ -6,14 +6,13 @@
 start by calling the initialization routine  \FUNC{shmem\_init}
 \footnote{\textbf{start\_pes} has been deprecated as of Specification 1.2}
 before using any of the other \openshmem library routines.  An \openshmem
-program finishes execution by returning from the main routine or when any PE
-calls \FUNC{shmem\_global\_exit}. When returning from main, \openshmem must
+program concludes its use of the \openshmem library when all \acp{PE} call
+\FUNC{shmem\_finalize} or when any \ac{PE} calls \FUNC{shmem\_global\_exit}.
```

Comment:
Split into two sentences:  "In addition, the program can be aborted when any PE calls ```shmem_global_exit.```"